x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list_table.tsx

```diff
@@ -794,7 +796,34 @@ export const RulesListTable = (props: RulesListTableProps) => {
     ruleOutcomeColumnField,
   ]);
 
-  const allRuleColumns = useMemo(() => getRulesTableColumns(), [getRulesTableColumns]);
+  const columnsOverrideMap = useMemo(() => {
```

Comment:
We're you able to test this? I sorted threw it together pretty quickly haha

Should we have a unit test for this as well?

Reply:
I guess this is https://github.com/elastic/kibana/issues/142145 right? I haven't tested anything related to this but as it doesn't break anything right now, what about adding it as it is and come back to finish it in another task?

Reply:
let's just write an issue and disable the props `columns` and come back to it

Reply:
I removed it here https://github.com/elastic/kibana/pull/147014/commits/7d240fa2bbc377cec5491e03537ba6b0d8e478b8 so I hope it's easy to recover